CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # vpqr Changelog
 
+## 2.2.0 -
```

Comment:
Don't forget to add a date.

Reply:
The date is only added when the package is published.